src/main/java/mat/client/shared/MatContext.java

```diff
@@ -1530,8 +1531,12 @@ public void setPopulationBasisList(ListBox listBox, List<String> list, String de
     public List<String> getPatientBasedIndicatorOptions(String measureScoringMethod) {
         List<String> patientBasedList = new ArrayList<>();
         patientBasedList.add("No");
-        if (!MatConstants.CONTINUOUS_VARIABLE.equalsIgnoreCase(measureScoringMethod)) {
+        if (!ModelTypeHelper.isFhir(getCurrentMeasureModel())) {
             patientBasedList.add("Yes");
+        } else {
+            if (!MatConstants.CONTINUOUS_VARIABLE.equalsIgnoreCase(measureScoringMethod)) {
+                patientBasedList.add("Yes");
+            }
         }
```

Comment:
Suggest combining into single `else if` instead of nesting. Simply removes a linebreak and a couple of square brackets when reading through the code.
```suggestion
        } else if (!MatConstants.CONTINUOUS_VARIABLE.equalsIgnoreCase(measureScoringMethod)) {
                patientBasedList.add("Yes");
        }
```